### PR TITLE
feat: remove target, platform from connector schema and add id to metadata

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

1. Remove `target` and `platform` from connector schema and add `id` to metadata.
2. Unwrap ConnectorMatata in ConnectorDTO.

## Linear Issue Reference

LOG-2444
LOG-2483

## Testing

Passed UTs.

### CODE REVIEW

File: packages/connector-alipay/src/constant.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ const readmeContentFallback = 'Please check README.md file directory.';
 const configTemplateFallback = 'Please check config-template.md file directory.';
 
 export const defaultMetadata: ConnectorMetadata = {
+  id: 'alipay_web',
```
Comment (gao-sun) [inline packages/connector-alipay/src/constant.ts:26]: ditto

**Review by gao-sun — COMMENTED** [review]
overall looks good, wait for another stamp

**Review by wangsijie — APPROVED** [review]
Overall LGTM, please resolve existing conversations.

File: packages/connector-alipay-native/src/index.ts [hunk 2]
```diff
@@ -37,15 +37,15 @@ import {
...
+import { signingParameters } from './utils';
+import type { SigningParameters } from './utils';
 
 export type { AlipayNativeConfig } from './types';
 
 export class AlipayNativeConnector implements SocialConnector {
   public metadata: ConnectorMetadata = defaultMetadata;
 
-  private readonly signingPamameters: SigningPamameters = signingPamameters;
+  private readonly signingParameters: SigningParameters = signingParameters;
```
Comment (demonzoo) [inline packages/connector-alipay-native/src/index.ts:48]: Do we need to explicitly declare the type?

**Pull request merged**